keps/sig-node/4004-deprecate-kube-proxy-version/kep.yaml

```diff
@@ -0,0 +1,41 @@
+title: Deprecate status.nodeInfo.kubeProxyVersion field
+kep-number: 4004
+authors:
+  - "@HirazawaUi"
+owning-sig: sig-node
+participating-sigs: []
+status: provisional
+creation-date: 2023-05-15
+reviewers:
+  - "@danwinship"
+  - "@thockin"
+approvers:
+  - "@sig-node-leads"
+
+see-also: []
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.28"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.28"
+  beta: "v1.29"
+  stable: "v1.30"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: kubeProxyVersion
```

Comment:
feature gates start with a capital letter, and should be more verbose than that. Maybe `DisableNodeKubeProxyVersion`.

Reply:
DisableNodeKubeProxyVersion used to be one of the options I was thinking about, and it would really be more relevant (although I discarded it at the last minute....)